### ISSUE TITLE
Error when use fixed.version

### ISSUE THREAD

**Issue by abel533** [spring-projects/spring-boot#4865]
When I config application as the following:
```
spring.resources.chain.strategy.fixed.enabled=true
spring.resources.chain.strategy.fixed.paths=/js/**,/v1.0.0/**
spring.resources.chain.strategy.fixed.version=v1.0.0
```
I use freemarker,the final source url like below:
```
<script type="text/javascript" src="/v1.0.0/js/index.js"></script>
```

If remove `/v1.0.0/**` from `paths`,the static source can't find `/v1.0.0` path.

I think, if I config the `fixed.version`,it should auto config to `fixed.paths`.This will solve the problem.

**Comment by bclozel** [comment]
This has been fixed with [SPR-13883](https://jira.spring.io/browse/SPR-13883) and will be available in Spring Framework 4.2.5.

@snicoll, I'm resolving and moving this issue to the 1.3.3 milestone, let me know if that was not the right place.

**Comment by snicoll** [comment]
Thanks Brian, I have created #5020 so that we make sure to upgrade to 4.2.5.
